### ISSUE TITLE
Charging timer fails to start on delayed start, after 40kwh swap [ZE0 2012]

### ISSUE THREAD

**Issue by Toby-TheEnglishMechanic** [dalathegreat/BatteryUpgrade-UserManual#14]

**Comment by dalathegreat** [comment]
Hi @Toby-TheEnglishMechanic , which SW version did this issue occur with?

**Comment by Toby-TheEnglishMechanic** [comment]
Hi @dalathegreat , this is on the v2.33, as delivered - I'm still awaiting my thingy to update the software ! 
I f its still relevant to current software release, I also found that once the car is switched off & has shut down fully, plugging in the L2 charger during the pre-set charging period, it also then fails to charge. Switching on & off again, to 're-energise' everything, allows the charge to start ..... sorry if the new SW already sorted all of this ;-)

**Comment by dalathegreat** [comment]
Yeah let's get you upgraded to v2.41 asap, this fixes the charge timer time estimates, it will behave very badly if you try to use timers without the latest version!

**Comment by rjsc** [comment]
v2.45 is also failing to charge with the timer in a 2012 ZE0 with a 30kWh battery.
I programmed my usual 80% charge without any starting time, just with the finishing  time.
The 'charging timer over-ride' button also fails as described in issue #15.
The only way to charge
